<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSingleCellPvar</class>
 <widget class="QDialog" name="AddSingleCellPvar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="ionChannelTypeLabel">
       <property name="text">
        <string>Ion Channel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="ionChannelType"/>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="randomize">
       <property name="text">
        <string>Random Values</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="normalDist">
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Nor&amp;mal Distribution</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="lognormalDist">
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Lognormal Distribution</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="meanLabel">
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mean (μ)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="mean">
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="stdDevLabel">
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Standard Deviation (σ)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="stdDev">
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="startValLabel">
       <property name="text">
        <string>Start Value</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="startVal">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="incAmtLabel">
       <property name="text">
        <string>Increment Amount</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="incAmt">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>+ Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>AddSingleCellPvar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
